### DISCUSSION TITLE
Ability to disable a camera processing via API

### DISCUSSION THREAD

**Discussion by mmaibaum** [blakeblackshear/frigate#1733] [Ideas]
For a couple of indoor cameras I've got I'd like to be able to only activate the cameras when we are out, I can switch them off but then frigate fills the log with (understandable) connection errors. It'd be great to be able to have a way to programatically 'turn off' a camera so frigate stops processing the feed entirely (I can turn off the camera itself in other ways). Currently the only way I can see to do this is to have two frigate configs and have some scripting to switch between them - not disastrous but a bit clumsy. I know you can turn off clips, detect, snapshot (though apparently not record - though wouldn't be surprised if that changed in 0.9?) via MQTT, something similar that just works on the camera object itself would be awesome... 

Anyone else think that would be useful or should I just resort to some config file hacking and bouncing the container...?

**Comment by mmaibaum** [comment]
somewhat related... https://github.com/blakeblackshear/frigate/issues/1729

**Comment by blakeblackshear** [comment]
There are several requests for this already. It's on the list for future enhancements.

**Reply by mmaibaum** [reply]
ah nice, thank you for the reply. No promises on timescales but (probably during the winter evenings when there isn't much else to do) I'll take a look at what it might take to understand enough of the internals to do and see if I can make some time. 

Cheers

Michael

**Comment by computerconsultingpeople** [comment]
Not trying to hijack this thread, I wonder if my requirement is similar. I have several PIR sensors that I use in conjunction with frigate  for example binary_sensor.motionsensor_003500130e_breezeway_occupancy , I want to tell frigate to record a snapshot for a particular camera based on this trigger. Is this possible ? Am i making any sense ?

**Reply by MrAnonymous722** [reply]
I actually think home assistant has this functionality built in, as long as you have a stream going thru HA for that camera. [I think this might work for you](https://www.home-assistant.io/integrations/camera/#services)

**Comment by ldotlopez** [comment]
Hi,

Any updates on this? I think it would be a good addition to Frigate.

I have a bunch of cameras at home that are automated to turn on/off based on some conditions (mainly if someone is at home).

Because of this, while cameras are off, Frigate's logs are flooded with errors, and lots of ffmpeg processes are spawned (and die) without a reason.

Anyway, thanks for all your work on the great software that is Frigate

**Comment by aztazt** [comment]
Hi,

Sorry to "necroposting", but I have the exact same request as @ldotlopez and I can't find a way to do it with MQTT.
Has this feature already been implemented and have I missed it in the documentation?
How guys do you do when you turn off cameras to avoid all the ffmpeg errors in Frigate?

Thank you @blakeblackshear for this masterpiece of software by the way.
